docs/source/library/cost_aware_synthesis.rst

```diff
@@ -0,0 +1,5 @@
+cost_aware_synthesis
+========================
```

Comment:
This is not an appropriate rst header definition. The "="s should always match the length of the header.

Reply:
done 👍🏻